packages/sw-background-sync-queue/src/index.js

```diff
@@ -0,0 +1,31 @@
+/*
+ Copyright 2016 Google Inc. All Rights Reserved.
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+ http://www.apache.org/licenses/LICENSE-2.0
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+ */
+
+/**
+ * sw-background-sync-queue Module
+ *
+ * @module sw-background-sync-queue
+ */
+
+import requestManager from './lib/requestManager';
+import queue from './lib/queue';
+
+function initialize(config, callbacks) {
```

Comment:
after removing the singleton nature of the modules, this initialize method no more is just a wrapper